app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}. {self.nickname} has a "
+                f"hummer of the {self._hummer_level} level")
+
+    def get_rating(self) -> int:
+        level_up = self._hummer_level + 4
+        return level_up
```

Comment:
The logic for calculating the rating in `get_rating` by simply adding 4 to `hummer_level` seems arbitrary. Ensure this logic aligns with the task requirements or provide a rationale for this calculation if it's intentional.